### ISSUE TITLE
Error when loading models with version 0.6.0

### ISSUE THREAD

**Issue by artidoro** [explosion/spacy-transformers#181]
I don't seem to be able to load the BERT models with the new release of spacy-transformers. I would have expected to do so without errors. The same commands work fine with version 0.5.2.

On a clean conda environment with python 3.8 I ran:
```
pip install spacy-transformers
python -m spacy download en_trf_bertbaseuncased_lg
```
Then in python
```python
>>> import spacy
>>> spacy.load('en_trf_bertbaseuncased_lg')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy/__init__.py", line 30, in load
    return util.load_model(name, **overrides)
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy/util.py", line 164, in load_model
    return load_model_from_package(name, **overrides)
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy/util.py", line 185, in load_model_from_package
    return cls.load(**overrides)
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/en_trf_bertbaseuncased_lg/__init__.py", line 12, in load
    return load_model_from_init_py(__file__, **overrides)
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy/util.py", line 228, in load_model_from_init_py
    return load_model_from_path(data_path, meta, **overrides)
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy/util.py", line 211, in load_model_from_path
    return nlp.from_disk(model_path, exclude=disable)
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy/language.py", line 947, in from_disk
    util.from_disk(path, deserializers, exclude)
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy/util.py", line 654, in from_disk
    reader(path / key)
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy/language.py", line 941, in <lambda>
    deserializers[name] = lambda p, proc=proc: proc.from_disk(
  File "pipes.pyx", line 228, in spacy.pipeline.pipes.Pipe.from_disk
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy/util.py", line 654, in from_disk
    reader(path / key)
  File "pipes.pyx", line 219, in spacy.pipeline.pipes.Pipe.from_disk.load_model
  File "/home/artidoro/anaconda3/envs/spacy-transformers/lib/python3.8/site-packages/spacy_transformers/_tokenizers.py", line 60, in from_bytes
    setattr(self, field, msg[field])
KeyError: 'init_kwargs'

```
The packages currently installed are (output of pip list):
```
Package                   Version            
------------------------- -------------------
blis                      0.4.1              
boto3                     1.13.16            
botocore                  1.16.16            
catalogue                 1.0.0              
certifi                   2020.4.5.1         
chardet                   3.0.4              
click                     7.1.2              
cymem                     2.0.3              
docutils                  0.15.2             
en-trf-bertbaseuncased-lg 2.2.0              
filelock                  3.0.12             
ftfy                      5.7                
future                    0.18.2             
idna                      2.9                
jmespath                  0.10.0             
joblib                    0.15.1             
murmurhash                1.0.2              
numpy                     1.18.4             
pip                       20.0.2             
plac                      1.1.3              
preshed                   3.0.2              
python-dateutil           2.8.1              
pytokenizations           0.4.10             
regex                     2020.5.14          
requests                  2.23.0             
s3transfer                0.3.3              
sacremoses                0.0.43             
sentencepiece             0.1.91             
setuptools                46.4.0.post20200518
six                       1.15.0             
spacy                     2.2.4              
spacy-transformers        0.6.0              
srsly                     1.0.2              
thinc                     7.4.0              
tokenizers                0.5.2              
torch                     1.5.0              
torchcontrib              0.0.2              
tqdm                      4.46.0             
transformers              2.5.1              
urllib3                   1.25.9             
wasabi                    0.6.0              
wcwidth                   0.1.9              
wheel                     0.34.2 
```

**Comment by kieran-lemayai** [comment]
Upvoting. I'm trying to load `en_trf_distilbertbaseuncased_lg` and getting the same error.

**Comment by adrianeboyd** [comment]
Due to changes in `transformers` the same models will not be compatible for 0.5 and 0.6. We'll release new models for along with the final 0.6.0 release.

In the meanwhile, you can always create a model locally using the `examples/init_model.py` script.

**Comment by adrianeboyd** [comment]
Sorry, it turns out that I was confused about the status of the 0.6.0 release. We got a bit out-of-sync internally in terms of coordinating the various steps related to the release of 0.6.x and I thought 0.6.0 was still in more of a draft state than it is, when it's published on pypi and available more widely than I though.

I'd recommend downgrading to 0.5.2 for the next few days until we get everything sorted out.

**Comment by adrianeboyd** [comment]
We just released 0.6.1 which fixes these inconsistencies and released updated compatible versions of models like `en_trf_distilbertbaseuncased_lg` along with it that you can download with `spacy download`.
